README.md

```diff
@@ -129,7 +129,10 @@ To use your own word embeddings or vocabulary set, specify the following paramet
     language 0.8667483925819397 5.001194953918457 ...
     ```
 - **vocab_file**: set the file path to a predefined vocabulary set that contains lines of words.
-
+    ```=
+    team
+    language
```

Comment:
not sure if readers think team and language have special meaning?
maybe 
a
an
the
...
is better?